test/distributed/test_functional_api.py

```diff
@@ -360,13 +375,15 @@ def test_reduce_scatter_into_tensor_coalesced(self, device):
         self.assertEqual(torch.tensor([8], device=device), res[1])
 
 
+@skipIfHpu
```

Comment:
Why do we want to skip here?

Reply:
You are right, it is not needed here. I have removed it in the latest commit.
Thanks!